modules/mixins/browser-state.js

```diff
@@ -59,6 +71,17 @@ var BrowserStateMixin = {
     });
   },
 
+  radiumGlobalMouseUp: function (ev) {
+    if (ev.toElement === this.getDOMNode()) {
```

Comment:
Given the desired behavior, is there any reason we couldn't remove this check *and* remove the mouseUp behavior in `browser-state.js`? https://github.com/FormidableLabs/radium/blob/master/modules/mixins/browser-state.js#L54-L60

Then the global mouseUp could handle both cases.